lib/worker/batcher.ex

```diff
@@ -1039,15 +1039,17 @@ defmodule BorsNG.Worker.Batcher do
         :sufficient
       end
 
+    pr = GitHub.get_pr!(repo_conn, patch.pr_xref)
+
     Logger.info(
-      "Code review status: Label Check #{passed_label} Passed Status: #{no_error_status and no_waiting_status and no_unset_status} Passed Review: #{passed_review} CODEOWNERS: #{code_owners_approved} Passed Up-To-Date Review: #{passed_up_to_date_review}"
+      "Code review status: Label Check #{passed_label} Passed Status: #{no_error_status and no_waiting_status and no_unset_status} Passed Review: #{passed_review} CODEOWNERS: #{code_owners_approved} Passed Up-To-Date Review: #{passed_up_to_date_review} Not Draft: #{!pr.draft}"
     )
 
     case {passed_label, no_error_status, no_waiting_status, no_unset_status, passed_review,
-          code_owners_approved, passed_up_to_date_review} do
-      {true, true, true, true, :sufficient, true, :sufficient} -> {:ok, toml.max_batch_size}
-      {false, _, _, _, _, _, _} -> {:error, :blocked_labels}
-      {_, _, _, _, _, _, _} -> :waiting
+          code_owners_approved, passed_up_to_date_review, !pr.draft} do
+      {true, true, true, true, :sufficient, true, :sufficient, true} -> {:ok, toml.max_batch_size}
```

Comment:
Therapist: big tuple can't hurt you

Big tuple:

Reply:
LOL